.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig documentation: https://editorconfig.org
+
+root = true
+
+# Default settings applied to every file type.
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 80
```

Comment:
We allow wider lines of code, we just wrap comments at 80 columns.

Reply:
It seems `max_line_length` doesn't formally distinguish code vs comments, so I can leave this directive out of the file for now.